lib/Components/badges.css

```diff
@@ -0,0 +1,112 @@
+.badge-container {
+    position: relative;
+}
+
+.badge {
+    border-radius: 50%;
+    position: absolute;
+    border: 2px solid var(--white-color);
+} 
+
+.badge-lg {
+    width: 2rem;
+    height: 2rem;
+}
+
+.badge-md {
+    width: 1.7rem;
+    height: 1.7rem;
+}
+
+.badge-sm {
+    width: 1.3rem;
+    height: 1.3rem;
+}
+
+.badge-xsm {
+    width: 14px;
+    height: 14px;
+}
+
+.avatar-lg-size > .badge {
+    bottom: 1rem;
+    right: 12px;
+}
+
+.avatar-md-size > .badge {
+    bottom: 10px;
+    right: 8px;
+}
+
+.avatar-sm-size > .badge {
+    bottom: 5px;
+    right: 4px;
+}
+
+.avatar-xsm-size > .badge {
+    bottom: 1px;
+    right: 1px;
+}
+
+.status-online-badge {
+    background: var(--green-color);
+}
+
+.status-offline-badge {
+    background: var(--light-gray-color)
+}
+
+.status-dnd-badge {
+    background: var(--red-color);
+}
+
+.status-idle-badge {
+    background: var(--darker-yellow-color);
+}
+
+.badge-container > i {
+    font-size: 2rem;
+}
```

Comment:
use shorter classname for short typing

Reply:
But shorter than this will not make sense to user what classes are used for and what are they means